README.md

```diff
@@ -1,2 +1,28 @@
-# portfolio_website
-portfolio wesite showcasing my work
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#Hello microverse
+
+This repository is an intro project to microverse with HTML,CSS and Linters. 
+
+
+![screenshot](./app_screenshot.png)
+
+## Built With
+HTML
+CSS
+JavaScript
+### Setup
+clone to you local machine
+create a directory on a your local machin.
+open your terminal and type: git clone https://github.com/chidiebereojingwa/Hello_Microverse.git
+### Usage
+Clone project to your local machine 
+run the index.html
```

Comment:
- [ ] Please, update your README with the missing sections e.g `LICENCE` section. I encourage you to use the README template provided by Microverse and customize the content.
- [ ] Please improve the Pull Request message. Typically, it should be the summary of the features you implemented.